lib/logstorage/filter_time_offset.go

```diff
@@ -0,0 +1,36 @@
+package logstorage
+
+import (
+	"fmt"
+
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/prefixfilter"
+)
+
+// filterTimeOffset has different behaviour, depends on query context:
+//
+// If there are more than one time filter in a single 'and' group,
+// so time offset should be applied to all time filters in the group.
+// For example, '_time:1h and _time:offset 1h' should be merged into '_time:2h'
```

Comment:
This is unexpected. I'd expect that `_time:1h and _time: offset 1h` should be converted into `_time:1h offset 1h`, e.g. it must select logs on the time range `[now-2h, now-1h]`.